libs/auth/src/cryptography.ts

```diff
@@ -550,3 +550,52 @@ export async function signMessage({
     : [scriptPath, scriptInput];
   return runCommand(command, { stdin: message });
 }
+
+/**
+ * Generates a random key of the specified length using OpenSSL. The key is
+ * returned in base64 encoding.
+ */
+export async function generateRandomKey(length: number): Promise<string> {
```

Comment:
suggest renaming this `generateRandomAes256Key` and bundling 256 into the function. Separating the logic of key length from algorithm is not useful and in the future could lead to badness.

Reply:
Suggestion implemented